### ISSUE TITLE
archievments crash

### ISSUE THREAD

**Issue by TamedYT** [ChristopherHX/mcpelauncher-manifest#206]
every time I try to open achievements my game crash

**Comment by Haus3master** [comment]
Yes, I have noticed that, after the implementation of the new achievements UI by mojang, the game began to crash upon trying to access it, the owner probably wont fix it because major UI changes will be rolled out in the future and developing to fix current issues might slow down future development, and anyways, 1.17 is not too far ahead, so, I assume that the owner will update it when a stable version of the game is released with big changes...

**Comment by TamedYT** [comment]
But I'm kinda playing on a release or a hotfix at least
